sys/arm64/include/param.h

```diff
@@ -114,7 +114,7 @@
 
 #ifndef KSTACK_PAGES
 #ifdef __CHERI_PURE_CAPABILITY__
-#define	KSTACK_PAGES	5	/* pages of kernel stack (with pcb) */
+#define	KSTACK_PAGES	8	/* pages of kernel stack (with pcb) */
```

Comment:
Should we do the same for RISC-V?

Hmm, right now we use 5 pages (instead of 4) for RISC-V purecap kernels from commit 4d0ede3132ec3.